src/boot/chid.c

```diff
@@ -73,6 +73,14 @@ static void smbios_info_populate(SmbiosInfo *ret_info) {
         ret_info->smbios_fields[CHID_SMBIOS_FAMILY] = smbios_to_hashable_string(raw.family);
         ret_info->smbios_fields[CHID_SMBIOS_BASEBOARD_PRODUCT] = smbios_to_hashable_string(raw.baseboard_product);
         ret_info->smbios_fields[CHID_SMBIOS_BASEBOARD_MANUFACTURER] = smbios_to_hashable_string(raw.baseboard_manufacturer);
+
+        EFI_STATUS status = edid_get_panel_id(&ret_info->smbios_fields[CHID_EDID_PANEL]);
+#if SD_BOOT
+        if (EFI_STATUS_IS_ERROR(status))
+                log_warning_status(status, "Failed to get EDID Panel ID: %m");
+#else
+        (void) status;
```

Comment:
Please add a short comment about that this is for suppressing unused variable compiler warning.